assets/css/edd-admin.css

```diff
@@ -3706,3 +3685,78 @@ div:not(#edd-item-tab-wrapper) + #edd-item-card-wrapper .item-section { /** [1]
 	font-style: italic;
 	width: 150px;
 }
+
+/**
+ * Forms
+ * ---------------------------------------------------------- */
+
+/**
+ * Form Group
+ *
+ * <div class="edd-form-group">
+ *   <label class="edd-form-group__label">Label</label>
+ *   <div class="edd-form-group__control">
+ *   	<input type="text" class="edd-form-group__input" />
+ *   </div>
+ *   <p class="edd-form-group__help">Help</p>
+ * </div>
+ *
+ *
+ * <fieldset class="edd-form-group">
+ *   <legend class="edd-form-group__label">Label</label>
+ *
+ *   <div class="edd-form-group__control">
+ *   	<input type="radio" class="edd-form-group__input" />
+ *   	<label>Radio 1</label>
+ *   </div>
+ *
+ *   <div class="edd-form-group__control">
+ *   	<input type="radio" class="edd-form-group__input" />
+ *   	<label>Radio 2</label>
+ *   </div>
+ * </fieldset>
+ *
+ */
+.edd-form-group {
+	margin-bottom: 12px;
+}
+
+.edd-form-group:last-of-type {
+	margin-bottom: 0;
+}
+
+.edd-form-group__label {
+	display: block;
+	margin-bottom: 4px;
```

Comment:
Looking through some Gutenberg/block editor styles they have both `4px` and `8px` instances. Chatter around the "G2" UI updates for Gutenberg are to add more breathing room around a lot of things, so I'm fine moving to `8px` for now.

Since we can now easily update it in the future, why not experiment?! 😄